requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+check-manifest
+pytest
+pytest-xdist
+twine
+wheel
```

Comment:
Some extras dev tools that we only use for testing.

Note that `pip` install instructions and `conda` instructions are using the same file (`requirements.txt`).